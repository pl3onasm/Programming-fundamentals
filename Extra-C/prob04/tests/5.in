10896 25999