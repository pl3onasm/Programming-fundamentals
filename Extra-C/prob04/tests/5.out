16513: 100000010000001
16833: 100000111000001
17057: 100001010100001
17377: 100001111100001
17553: 100010010010001
17873: 100010111010001
18097: 100011010110001
18417: 100011111110001
18569: 100100010001001
18889: 100100111001001
19113: 100101010101001
19433: 100101111101001
19609: 100110010011001
19929: 100110111011001
20153: 100111010111001
20473: 100111111111001
20613: 101000010000101
20933: 101000111000101
21157: 101001010100101
21477: 101001111100101
21653: 101010010010101
21973: 101010111010101
22197: 101011010110101
22517: 101011111110101
22669: 101100010001101
22989: 101100111001101
23213: 101101010101101
23533: 101101111101101
23709: 101110010011101
24029: 101110111011101
24253: 101111010111101
24573: 101111111111101
24707: 110000010000011
25027: 110000111000011
25251: 110001010100011
25571: 110001111100011
25747: 110010010010011
